server/controllers/Shred.controller.js

```diff
@@ -1,14 +1,30 @@
-// uncomment this
-// import Shred from '../models/Shred.model';
+import Shred from '../models/Shred.model';
+/*create new shred. */
+function create(req, res, next) {
+  const shred = new Shred({
+    ownerId: req.body.ownerid,
+    shredSize: req.body.shredsize
+  });
```

Comment:
supposed to use `req.params.ownerId` and `req.params.shredSize` instead of accessing `body`